cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterConfig.java

```diff
@@ -48,6 +52,8 @@
   private static final Integer DEFAULT_CRUISE_CONTROL_METRICS_TOPIC_NUM_PARTITIONS = -1;
   private static final boolean DEFAULT_CRUISE_CONTROL_METRICS_TOPIC_AUTO_CREATE = false;
   private static final Short DEFAULT_CRUISE_CONTROL_METRICS_TOPIC_REPLICATION_FACTOR = -1;
+  private static final String DEFAULT_CRUISE_CONTROL_METRICS_TOPIC_CLEAN_UP_POLICY = "delete";
```

Comment:
I don't think this should be configurable -- i.e. we do not support any other configuration, let's avoid exposing it.